### PR TITLE
Fix funny dot bug

### PR DESCRIPTION
**Pull request by hun1ahpu**
Fix for #27718

### CODE REVIEW

File: src/vs/base/common/labels.ts [hunk 1]
```diff
@@ -134,7 +134,7 @@ export function shorten(paths: string[]): string[] {
 		let path = paths[pathIndex];
 
 		if (path === '') {
-			shortenedPaths[pathIndex] = '.';
+			shortenedPaths[pathIndex] = '.' + nativeSep;
```
Comment (bpasero) [inline src/vs/base/common/labels.ts]: @hun1ahpu 💄 

`.${nativeSep}`

**Review by bpasero — COMMENTED** [review]
👍

**Comment by hun1ahpu** [comment]
Not sure why travis is failing

**Comment by bpasero** [comment]
Thanks!

**Pull request merged**